storage/rocksdb/ha_rocksdb.cc

```diff
@@ -44,7 +44,6 @@
 #include "rocksdb/slice_transform.h"
 #include "rocksdb/utilities/checkpoint.h"
 #include "rocksdb/utilities/convenience.h"
-#include "rocksdb/utilities/flashcache.h"
```

Comment:
This and everything below should have been merged to 5.7 in their respective PRs, no?